tv_apps/browser/locales/browser.en-US.properties

```diff
@@ -98,14 +91,17 @@ save-video=Save Video
 save-audio=Save Audio
 
 # Toolbar
-sign-in-to-sync=Sign in to Sync
 pocket-list=Pocket List
 
-# Firefox Account
-fxa-resend-alert=A verification link has been sent to {{email}}.
-fxa-sign-out=Sign Out
-fxa-sign-out-confirm-msg=Are you sure? Firefox will stop syncing with your account, and delete related browsing data on this TV.
-fxa-sign-out-alert=You have signed out successfully.
+# Firefox Sync
+fxsync-create-account-or-sign-in=Create account or Sign in
```

Comment:
The button name is changed to ```Sign In``` as discussed. https://drive.google.com/drive/folders/0B4dMhI4hp32OanhJQ1NnMVMydTA